src/screen.c

```diff
@@ -10397,7 +10397,8 @@ draw_tabline(void)
 	FOR_ALL_TABPAGES(tp)
 	    ++tabcount;
 
-	tabwidth = (Columns - 1 + tabcount / 2) / tabcount;
+	if (tabcount > 0)
+	    tabwidth = (Columns - 1 + tabcount / 2) / tabcount;
```

Comment:
I don't think that tabcount can possibly be <= 0, unless there is a bug.
So this should ideally be an assertion. 
 
Vim source code does not use assertions though.
I think it would be good to use assertions.
It could help detecting more errors when fuzzing among other things.